src/js/components/stock-movement-wizard/AddItemsPage.jsx

```diff
@@ -861,9 +863,17 @@ class AddItemsPage extends Component {
    */
   transitionToNextStep(status) {
     const url = `/openboxes/api/stockMovements/${this.state.values.stockMovementId}/status`;
-    const payload = { status };
+    let payload = [];
 
-    return apiClient.post(url, payload);
+    if (this.state.values.statusCode === 'CREATED') {
+      payload = { status };
+    }
+
+    if (payload.length) {
```

Comment:
I guess you can get rid of this if. I can have const payload = { status } as before and if statusCode === created, then do post. No need for this check, because you dont use payload if statusCode !== created.